<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenWidget</class>
 <widget class="QWidget" name="ScreenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Setup</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelTwinView">
     <property name="text">
      <string>Twinview:</string>
     </property>
     <property name="buddy">
      <cstring>twinview_comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="twinview_comboBox">
     <item>
      <property name="text">
       <string comment="none means disabled twinview">none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="monitors are vertical aligned">vertical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="monitors are horizontal aligned">horizontal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTvResolution1">
     <property name="text">
      <string>Resolution 1:</string>
     </property>
     <property name="buddy">
      <cstring>spinBox_1_x</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KIntSpinBox" name="spinBox_0_x">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="xlabel1">
       <property name="text">
        <string comment="Divider for the Metamode Display resolution settings like 1280x1024">x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KIntSpinBox" name="spinBox_0_y">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTvResolution0">
     <property name="text">
      <string>Resolution 2:</string>
     </property>
     <property name="buddy">
      <cstring>spinBox_0_x</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KIntSpinBox" name="spinBox_1_x">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="xlabel2">
       <property name="text">
        <string comment="Divider for the Metamode Display resolution settings like 1280x1024">x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KIntSpinBox" name="spinBox_1_y">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="KComboBox" name="screenComboBox">
     <item>
      <property name="text">
       <string comment="none means no screen mapped">none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true" comment="Monitor number 1">1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true" comment="Monitor number 2">2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="mmonitor_checkBox">
     <property name="text">
      <string comment="enables multi monitor support if twinview is not enabled">on</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelMuliMonitor">
     <property name="text">
      <string>Multi Monitor:</string>
     </property>
     <property name="buddy">
      <cstring>mmonitor_checkBox</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelScreenN0">
     <property name="text">
      <string>Map to screen:</string>
     </property>
     <property name="buddy">
      <cstring>screenComboBox</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>twinview_comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>twinview_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screenComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mmonitor_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twinview_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>switchTwinView(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_0_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_1_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_0_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_1_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScreenWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>profileChanged()</slot>
  <slot>switchTwinView(int)</slot>
 </slots>
</ui>
